lib/bulk_test.go

```diff
@@ -45,10 +45,55 @@ var _ = Describe("bulk", func() {
 			sfServer.AppendHandlers(
 				CombineHandlers(
 					VerifyRequest("GET", "/services/async/45.0/job//batch/batch1/result/result1"),
-					RespondWith(400, "whoops"),
+					RespondWith(400, "<LoginFault><exceptionCode>Yo</exceptionCode></LoginFault>", XmlHeaders),
```

Comment:
I don't think we can get a `LoginFault` here.  If the session token is invalid, we'll get:
```
<?xml version="1.0" encoding="UTF-8"?>
<error
xmlns="http://www.force.com/2009/06/asyncapi/dataload">
  <exceptionCode>InvalidSessionId</exceptionCode>
  <exceptionMessage>Invalid session id</exceptionMessage>
</error>
```

Reply:
Updated. Because the root element name in XML isn't used by Go xml unmarshaling, it didn't effect the correctness of the code.